eng/common/TestResources/build-test-resource-config.yml

```diff
@@ -49,5 +49,13 @@ steps:
         ${{ convertToJson(parameters.SubscriptionConfigurationFilePaths) }}
       '@.Trim()
 
+      # When variable groups are not added to the pipeline, secret references like
```

Comment:
Should we do this in the script and apply it to all inputs?

Reply:
You mean check if it's a json object (e.g. starts with `[` or `{`)? That could work too, originally I was thinking this was just a pipeline specific problem hence why I surfaced the validation at this level, but I think both directions are reasonable.

Reply:
Yes just validate all 3 inputs for this issue in the script before calling convertfrom-json